client/components/swipeable/docs/example.jsx

```diff
@@ -0,0 +1,28 @@
+import { PureComponent } from 'react';
+import Swipeable from 'calypso/components/swipeable';
+
+export default class extends PureComponent {
```

Comment:
I think we can make this a functional component? We're usually refactoring class components when they don't need to be class components. And people tend to copy a lot from the examples 😉

Reply:
Good idea.